assets/js/googlesitekit/notifications/components/Notification/index.js

```diff
@@ -36,10 +46,10 @@ export default function Notification( { id, className, children } ) {
 	// Track view once.
 	useEffect( () => {
 		if ( ! isViewedOnce && viewed ) {
-			trackEvents.view();
+			trackEvents.view( ...gaTrackingEventArgs );
```

Comment:
Thank you for making the suggested change, it looks great!

Now that I look at it, we’re passing an array which is then spread to access parameters, which I feel is error-prone as this is reliant on the order of arguments in the array.

As discussed internally, I feel using an object with strictly defined properties for arguments would be a nice improvement.

Reply:
Updated, resolving.